scripts/input/climate_assessment_openscm_run.py

```diff
@@ -18,8 +18,8 @@
 
 # TONN REMOVE start
 # If these are already set, it shouldn't override it. We actually may not want to have a default but just throw an error if not set
```

Comment:
So the code in the following lines actually overwrites the environment variables `MAGICC_EXECUTABLE_7` & `MAGICC_WORKER_ROOT_DIR`. If you want to achieve the behavior described in this comment, change it to

```
class EnvironmentError(Exception):
    pass

for env_var in ["MAGICC_EXECUTABLE_7", "MAGICC_WORKER_ROOT_DIR"]:
    if os.environ.get(env_var, '') == '':
        # If clause covers both cases in which the env var is not set at all 
        # as well as the case in which it is set to an empty string
        raise EnvironmentError(f"{env_var} does not exist")
```

Reply:
Nice, thanks!